Editor/Editors/CinemachineTrackedDollyEditor.cs

```diff
@@ -1,11 +1,14 @@
+using System;
 using System.Collections.Generic;
 using UnityEditor;
 using UnityEngine;
-
 namespace Cinemachine.Editor
 {
     [CustomEditor(typeof(CinemachineTrackedDolly))]
     [CanEditMultipleObjects]
+#if UNITY_2022_1_OR_NEWER
+    [Obsolete("CinemachineTrackedDolly has been deprecated. Add Splines package to your project, and use CinemachineSplineDolly instead.", false)]
+#endif   
```

Comment:
The reason why I added it, is because otherwise Unity is throwing warnings in the console complaining the CmTrackedDolly is deprecated. I could either add pragma suppression for each usage in this file or this one #if obsolete tag #endif.

Reply:
ok